modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/ProductOptionValueUtil.java

```diff
@@ -51,7 +51,8 @@ public class ProductOptionValueUtil {
 			cpDefinitionOptionValueRel =
 				cpDefinitionOptionValueRelService.
 					updateCPDefinitionOptionValueRel(
-						cpDefinitionOptionValueRel.getCPDefinitionOptionRelId(),
+						cpDefinitionOptionValueRel.
+							getCPDefinitionOptionValueRelId(),
```

Comment:
check if this change is to fix an error, looks like it is

Reply:
I can confirm. It was a bug solved